packages/hackathon/src/scenes/JudgingEditorScene/components/JudgingForm.tsx

```diff
@@ -71,111 +68,102 @@ export const JudgingForm: FC<JudgingFormProps> = ({ judging, readonly }) => {
+        <Form width="70vh">
+          <FieldSlider
+            detail="How well does the project accomplish its goal?"
+            description="Scale from 1 to 10 with 1 the lowest and 10 the highest"
+            label="Execution"
+            min={1}
+            max={10}
+            step={1}
+            onChange={onValueChange}
+            value={execution}
+            disabled={readonly}
+            name="execution"
+          />
+          <FieldSlider
+            detail="How complex is the project's functionality?"
+            description="Scale from 1 to 10 with 1 the lowest and 10 the highest"
+            label="Scope"
+            min={1}
+            max={10}
+            step={1}
+            onChange={onValueChange}
+            value={scope}
+            disabled={readonly}
+            name="scope"
+          />
+          <FieldSlider
+            detail="How novel, unique, or interesting is the project?"
+            description="Scale from 1 to 10 with 1 the lowest and 10 the highest"
+            label="Novelty"
+            min={1}
+            max={10}
+            step={1}
+            onChange={onValueChange}
+            value={novelty}
+            disabled={readonly}
+            name="novelty"
+          />
+          <FieldSlider
+            detail="How useful could the project be to the developer population at large?"
+            description="Scale from 1 to 10 with 1 the lowest and 10 the highest"
+            label="Impact"
+            min={1}
+            max={10}
+            step={1}
+            onChange={onValueChange}
+            value={impact}
+            disabled={readonly}
+            name="impact"
+          />
+          <strong>Total Score: {judging.score}</strong>
+          <FieldTextArea
+            name="notes"
+            resize="vertical"
+            label="Notes"
+            placeholder="Additional comments about this project (supports markdown)"
+            defaultValue={judging.notes}
+            onChange={onValueChange}
+            disabled={readonly}
+          />
+          <Space between>
+            <Space>
+              <ButtonOutline type="button" onClick={handleCancel}>
+                Return to judging
+              </ButtonOutline>
+              <Button type="submit" onClick={handleSave} disabled={readonly}>
+                Save judging
+              </Button>
             </Space>
-          </SpaceVertical>
-        </Tab2>
-        <Tab2 id="preview" label="Preview">
-          <JudgingView judging={judging} />
-        </Tab2>
-      </Tabs2>
-    </SpaceVertical>
+          </Space>
+        </Form>
+      </Tab2>
+      <Tab2 id="project" label="Project details">
+        <ProjectView
+          title={judging.$title}
+          description={judging.$description}
+          technologies={judging.$technologies}
+          members={judging.$members}
+          project_type={judging.$project_type}
+          contestant={judging.$contestant}
+        />
```

Comment:
This could have passed a composed `project` object compatible with the `Project` interface without having to change `ProjectView`, I think.

Reply:
Yeah the most comprehensive change would be reverting this change and removing Judging's generated fields. It's duplicated info that we have to worry about syncing (i'm not even sure if they're kept up to sync). Our app already contains projects/team members/etc, we should be exposing that info through redux/saga to react, and not duplicating it for convenience.